### DISCUSSION TITLE
我SOLO爆块了，但是XCH币已经一天了还没到账

### DISCUSSION THREAD

**Discussion by 13430662839** [Chia-Network/chia-blockchain#4915] [Support]
我在高度272765挖到了两个XCH，
奖励地址填写为xch1eg3pmlvqaj4p5228kaqed6dlrhg49hc8gc9t0qayckmfug9qduvqp0gnf2
但是现在已经一天了，怎么奖励还没到账呢？

**Comment by Plopsi** [comment]
https://www.chiaexplorer.com/blockchain/address/xch1eg3pmlvqaj4p5228kaqed6dlrhg49hc8gc9t0qayckmfug9qduvqp0gnf2 looks good for me (approved on 272.768), check if your wallet is sync (not only full_node but also your wallet)

![grafik](https://user-images.githubusercontent.com/17942998/118226383-2f2d1380-b487-11eb-929d-e554c40ef826.png)

or `chia wallet show `

**Reply by 13430662839** [reply]
![image](https://user-images.githubusercontent.com/84167842/118249471-0d0fb180-b4d8-11eb-842c-07ad6a89d828.png)
![image](https://user-images.githubusercontent.com/84167842/118249534-1ac53700-b4d8-11eb-84a2-f700b5ee5509.png)
my full node  and wallnet is sync

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
